### ISSUE TITLE
Increase file size limit for projects catboost, catboost-dev

### ISSUE THREAD

**Issue by dbakshee** [pypa/warehouse#6972]
**Please increase the file size limit from 60MB to 80MB.** 

For project [catboost-dev](https://pypi.org/project/catboost-dev) we have hit the size limit in a recent upload, and I suspect we'll hit it for the production project [catboost](https://pypi.org/project/catboost) in a coming release.

Typical size of catboost release is about 550MB, Release contains whl files for 3 OSes times python versions 2.7, 3.5, 3.6, 3.7. We also publish on conda.

Please increase index for PyPI. We do not use Test PyPI.

CatBoost is a machine learning library by Yandex, specifically it is gradient boosting on decision trees library. Its close alternatives/rivals are [LightGBM](https://pypi.org/project/lightgbm/) by Microsoft and [XGBoost](https://pypi.org/project/xgboost). From release to release the library steadily increased capabilities and now approached the limit of 60MB/file. We are going to add new functionality that will certainly increase the size and exceed the current limit.
